packages/eas-cli/src/submit/ios/IosSubmitter.ts

```diff
@@ -152,6 +171,10 @@ function formatServiceAccountSummary({ summary }: AscApiKeyResult): string {
       label: 'Key ID',
       value: keyId,
     },
+    {
+      label: 'Key Name',
+      value: name,
+    },
```

Comment:
Maybe we should display key name first, before other values - moving this to the top should do the job.